docs/performance.md

```diff
@@ -0,0 +1,14 @@
+# Performance Optimization
+
+## Filter Bypass
+
+The agent is written in Go allowing to set performance expectations pretty high.
+
+Also, one must take care of the HAProxy configuration when using the agent. The example provided in this
+repository allows to split the domains into two categories: the public domains and the ones requiring authentication.
+If correctly configured, only domains requiring authentication will trigger a call to the agent hence keeping the
+raw performance for public domains. The trick is the condition in the `try-auth` message in the spoe configuration.
+
+    event on-frontend-http-request if { hdr_beg(host) -i app1.example.com } || { hdr_beg(host) -i app2.example.com }
```

Comment:
Maybe mention that this can be done with maps as well? (if the list is becoming too large)

Reply:
Sure. I don't know very well the HAProxy configuration and have not digged into it much so I did not know it was possible. I will add a mention for that. Thanks for the suggestion.